client/main.cpp

```diff
@@ -0,0 +1,51 @@
+#include <boost/program_options.hpp>
+#include <iostream>
+#include "tcp_client.hpp"
+
+int main(int argc, const char *argv[])
+{
+    uint16_t port_num = 3333;
+    std::string ip_addr = "127.0.0.1";
```

Comment:
having these parameters read from a config file makes it nicer I think.

Reply:
Done.